examples/tools/spec_cache/content/cache.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
```

Comment:
You'll need to set execute permissions on this script before building the container, otherwise you'll get an error